src/test/groovy/com/compilercharisma/chameleonbusinessstudio/integration/UserControllerITSpec.groovy

```diff
@@ -26,7 +26,7 @@ class UserControllerITSpec extends BaseITSpec{
                 .withHeader("Authorization", equalTo(VENDIA_API_KEY))
                 .withHeader("Content-Type", equalTo("application/json"))
                 .withHeader("Accept", equalTo("application/json, application/graphql+json"))
-                .withRequestBody(equalTo("{\"query\":\"query { list_UserItems { _UserItems { _id email displayName appointments } } }\"}"))
+                .withRequestBody(equalTo("{\"query\":\"query { list_UserItems { _UserItems { _id email displayName role appointments } } }\"}"))
```

Comment:
oh was this test failing because of the addition to the role attribute? I forgot about having to update this unit test when I added the role attribute to this query lol

Reply:
Yeah exactly, this test was failing just because the query changed, apparently these tests are tightly coupled to whatever the query is, so if the query changes, the tests might fail